src/API/sensors/sensor.controller.js

```diff
@@ -0,0 +1,42 @@
+import Sensor from './sensor.model';
+
+function get(req, res, next) {
+    Sensor.get(req.params.id).then((sensor) => {
```

Comment:
You can omit parentheses in the arrow functions in case you have 1 argument only